.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules/
 /lib-es5/
+.idea
+.github
+dist/@starface/
```

Comment:
please don't edit this file. You can achieve this by editing your local ~/.gitignore file.